packages/teleterm/src/ui/DocumentTerminal/useDocumentTerminal.ts

```diff
@@ -81,7 +81,7 @@ async function initState(
 
   const clusterUri = routing.getClusterUri(doc);
```

Comment:
Could you update the call here so that `leafClusterId` and `rootClusterId` are explicitly read from doc so that the usage of those fields shows up when using TypeScript editor plugins?

Reply:
Done